lib/build.py

```diff
@@ -832,12 +844,14 @@ def __init__(self, *args):
 class I_from_(Instruction):
 
    __slots__ = ("alias",
+                "arg",
                 "base_image")
 
    def __init__(self, *args):
       super().__init__(*args)
-      self.base_image = ch.Image(ch.Image_Ref(ch.tree_child(self.tree,
-                                                            "image_ref")))
+      self.arg = self.options.pop("arg", None)
```

Comment:
Give a default of the empty dictionary and you won't need `if (... not None)` below.